.github/workflows/go-testing-ci.yml

```diff
@@ -72,16 +72,35 @@ jobs:
         uses: actions/setup-go@master
         with:
           go-version: "1.21"
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3 
       - name: Create k8s Kind Cluster
-        uses: helm/kind-action@v1.1.0
+        uses: helm/kind-action@v1.5.0
         with:
           cluster_name: "kind-cluster"
       - name: Run coverage
-        run: go test -run Integration ./... -race -coverprofile=coverage.txt -covermode=atomic -timeout 40m
+        run: |
+          cd mesheryctl;
+          make; 
+          ./mesheryctl system start -p docker;
+          ./mesheryctl system update -y;
+          ./mesheryctl system status -y;
+          ./mesheryctl system restart -y;
+          ./mesheryctl system check --preflight;  
+          ./mesheryctl system stop -y;
+          echo "Running Mesheryctl with Docker completed."
+        
+          ./mesheryctl system start -p kubernetes;
+          ./mesheryctl system update -y;
+          ./mesheryctl system status -y;
+          ./mesheryctl system restart -y;
+          ./mesheryctl system check --preflight;
+          ./mesheryctl system stop -y;
+          echo "Running Mesheryctl with Kubernetes completed."
```

Comment:
Addition of these specific e2e tests for running Meshery on docker and kubernetes platform through mesheryctl is great and we don't have such test already running.

Though I don't think we should be displacing the existent unit tests.

Reply:
> Addition of these specific e2e tests for running Meshery on docker and kubernetes platform through mesheryctl is great and we don't have such test already running.
> 
> Though I don't think we should be displacing the existent unit tests.

This is an integration test and it has the environment prepared for running the  mesheryctl system tests on docker and k8. There is a go code written for it and it doesn't work . So I implemented them here

Reply:
Other failing tests have been removed and some new ones have been created the ones removed / commented out need to be re implemented

Reply:
Thanks!